apiserver/pkg/manager/client_manager.go

```diff
@@ -10,6 +10,7 @@ import (
 
 type ClientManagerInterface interface {
 	ClusterClient() client.ClusterClientInterface
+	RayJobClient()  client.JobClientInterface
```

Comment:
`RayJobClient` -> `JobClient ` for consistency

Reply:
fixed